CP-CPS.md

```diff
@@ -332,7 +332,7 @@ See Section 2.2 of this document for Root and Subordinate CA certificate publica
 
 All Subscriber Certificates are made available to Subscribers via the ACME protocol.
 
-For each Subscriber Certificate issuance, ISRG signs a Precertificate and submits it to a selection of Certificate Transparency logs. Upon successful submission, ISRG attempts to issue a certificate that matches the Precertificate (per RFC 6962 Section 3.1) and embeds at least two of the resulting Signed Certificate Timestamps (SCTs). ISRG submits the resulting final certificate to a selection of Certificate Transparency logs on a best-effort basis.
+Prior to each Subscriber Certificate issuance, ISRG signs a corresponding Precertificate and submits it to a selection of Certificate Transparency logs. After each Subscriber Certificate issuance, ISRG submits resulting certificate to a selection of Certificate Transparency logs on a best-effort basis.
```

Comment:
"ISRG submits resulting certificate to..."

The word "the" was deleted here creating a grammar issue.

Reply:
Fixed